pyproject.toml

```diff
@@ -119,3 +119,7 @@ pythonpath = "."
 env = [
   "OPTINIST_DIR=studio/test_data",
 ]
+
+[tool.codespell]
+exclude = '^frontend/(build|node_modules)/|.*\.lock$'
+ignore-words = '.codespellignore'
```

Comment:
There should be empty line for each files.
This rule will be checked by existing pre-commit rule(end-of-file-fixer).
Maybe you forgot to run `pre-commit install`.

Reply:
The irony I am working on a pre-commit function! Okay, I'll install it